Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
```

Comment:
platform version should be specified and match the project deployment target

Reply:
+1

Reply:
Just wondering ... Is it necessary to also do **pod update** after the ios platform in podfile has been updated?

Reply:
Nope but its good to lock pods at some version so ur sure when somebody does pod install/pod update it doesnt break build

Reply:
Pod install is enough, check difference between pod install and pod update

Reply:
Yep, good point.

Reply:
Locked pod version,  specified deployment target